src/status_im/ui/components/qr_code_viewer/views.cljs

```diff
@@ -15,8 +16,12 @@
 
 (defn qr-code-viewer [{:keys [style hint-style footer-style footer-button value hint legend]}]
   (if value
-    (let [{:keys [width]} @(re-frame/subscribe [:dimensions/window])]
+    (let [{:keys [width]} @(re-frame/subscribe [:dimensions/window])
+          snt-amount @(re-frame/subscribe [:get-in [:my-profile/tribute-to-talk :snt-amount]])]
       [react/view {:style (merge styles/qr-code style)}
+       (when snt-amount
+         [react/view {:style {:margin-horizontal 16}}
+          [ttt/ttt-enabled-note]])
```

Comment:
ttt - maybe we could find better naming? because it looks like mistake, and doesn't give any additional info about this package , tr-to-talk or smth

Reply:
Renamed to `tribute-to-talk` and `tr-to-talk` for this particular `require`.